packages/ctoxml/ctoxml.4.1.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+synopsis: "Parses a C program into Cabs AST and dumps as an XML document"
+maintainer: ["Ivan Gotovchits <ivg@ieee.org>"]
+authors: ["Hugues Cassé <casse@irit.fr> et al"]
+license: "LGPL-2.0-only"
+tags: ["FrontC" "C" "parser" "XML"]
+homepage: "https://github.com/BinaryAnalysisPlatform/FrontC"
+bug-reports: "https://github.com/BinaryAnalysisPlatform/FrontC/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "2.7"}
+  "FrontC" {= "4.1.0"}
```

Comment:
```suggestion
  "FrontC" {= version}
```
If this package is meant to be released in sync with FrontC, this might get handy